src/stories/CardMeta.stories.js

```diff
@@ -132,18 +132,18 @@ export function OnlyCategoryAndTitle() {
     template: `
       <card-meta
         :category="series.title"
-        :title="event.eventTitle"
+        :title="event.title"
       />
   `,
   }
 }
 
-export function ShareButton() {
+export function FtvaShareButton() {
```

Comment:
@jendiamond This story isn't needed. The share functionality is coming the dropdown component, not card meta.